src/java.desktop/windows/classes/com/sun/java/swing/plaf/windows/WindowsIconFactory.java

```diff
@@ -294,8 +294,9 @@ public int getIconWidth() {
                     width = (int) ((float) width * d.width / d.height);
                 }
             } else {
-                width = UIManager.getInt("InternalFrame.titleButtonWidth") -2;
+                width = UIManager.getInt("InternalFrame.titleButtonHeight") - 2;
```

Comment:
```suggestion
                width = UIManager.getInt("InternalFrame.titleButtonHeight") -2;
```
I agree it looks better with the space on either side but it's consistently used without the space after `-`. So I'd rather keep the original formatting.